src/main/scala/com/mozilla/telemetry/pings/MainPing.scala

```diff
@@ -107,6 +133,26 @@ case class MainPing(application: Application,
     dynamicProcessEvents.filter(_.category == "normandy")
   }
 
+
+  override def pingLevelEvents: Seq[JObject] = {
```

Comment:
I think it makes a lot more sense to insert an event into the main ping events list and add an entry into each individual config re: whether they'd like to use this event or not. As it stands, other jobs using this code will *always* have this event, which is problematic for a number of reasons. For instance, in the case of devtools which looks through *all* main pings, _every single main ping will now fire this event_ whether that client used devtools or not.

Reply:
Makes total sense. I'll add something to the JSON schema to enable or disable this behavior.